proposals/dht-js.md

```diff
@@ -0,0 +1,106 @@
+# Effective content and service discovery in JS
+
+Authors: @jacobheun
+
+Initial PR: TBD
+
+## Purpose &amp; impact
+#### Background &amp; intent
+_Describe the desired state of the world after this project? Why does that matter?_
+<!--
+Outline the status quo, including any relevant context on the problem you’re seeing that this project should solve. Wherever possible, include pains or problems that you’ve seen users experience to help motivate why solving this problem works towards top-line objectives.
+-->
+
+**Current State**
+- JS projects rely on delegate and preload nodes to be able to interact with the live network
+- PL hosted delegate/preload nodes are often overloaded negatively impact performance of projects
```

Comment:
@gmasgras this sounds like something that is soluble from an infra perspective. Can you comment on our current preload node uptime/utilization?